msi/docs/build-msi-locally.md

```diff
@@ -0,0 +1,139 @@
+Should you ever need to rebuild a Jenkins MSI on your Windows machine, here is a way to do it.
+
+# Pre-requisites
+
+## Jenkins WAR file
+
+First of all, you should get the Jenkins war file that will be inside that MSI file.
+You can get it from the official Jenkins website or from the Jenkins update center.
+I will use the official Jenkins website in this article.
+
+Check [Jenkins download page](https://www.jenkins.io/download/) and download the latest weekly version of Jenkins for example.
+For me, it is [jenkins.war](https://get.jenkins.io/war/2.380/jenkins.war) version 2.380.
+
+Change the URL according to the version you want to download.
+
+## Git tool
+
+You have a few options here:
+* Install [Git for Windows](https://community.chocolatey.org/packages/git)
+* Use your IDE to clone the repository
+* Use [GitHub Desktop](https://desktop.github.com/)
+* Use [SourceTree](https://www.sourcetreeapp.com/)
+* Use Cygwin integrated [Git command](https://cygwin.com/packages/summary/git.html)
+* [...]
```

Comment:
Off topic? I would simply specify git as requirement and a link to the official website.

Reply:
Done. Thanks.